src/test/java/com/android/volley/toolbox/DiskBasedCacheTest.java

```diff
@@ -501,86 +503,87 @@ public void testCountingInputStreamByteCount() throws IOException {
     public void testEmptyReadThrowsEOF() throws IOException {
         ByteArrayInputStream empty = new ByteArrayInputStream(new byte[] {});
         exception.expect(EOFException.class);
-        DiskBasedCache.readInt(empty);
+        DiskBasedCacheUtility.readInt(empty);
```

Comment:
For tests which solely exercise methods in DiskBasedCacheUtility, we should move their tests to a new DiskBasedCacheUtilityTest rather than having them in here, since they aren't specific to DiskBasedCache any more.

Reply:
Fixed this

Reply:
Scott, did you upload a new DiskBasedCacheUtilityTest class for the tests you removed from here? I don't see one currently